pandas/io/parsers.py

```diff
@@ -2194,16 +2194,17 @@ def _handle_usecols(self, columns, usecols_key):
         usecols_key is used if there are string usecols.
         """
         if self.usecols is not None:
-            if any([isinstance(u, string_types) for u in self.usecols]):
+            if any([isinstance(usecol, string_types)
+                    for usecol in self.usecols]):
```

Comment:
is this needed to pass the tests? (I can't imagine we nowhere use `i` in a iteration) as I actually find the one-liner better